### PR TITLE
Add a Quick Settings Tile to control/monitor service

### PR DESCRIPTION
**Pull request by edgarogh**
Adds a Quick Settings Tile to quickly toggle the service state and monitor its state. Most context can be found in #135.

A lot of things in the PR may be imperfect. Feel free to ask for changes.

<p align="center">

</p>

### CODE REVIEW

**Review by slowscript — COMMENTED** [review]
Thanks for the PR! This is a nice feature. Code looks good to me.
Just some minor possible improvements, I might even do it myself if I get to this again soon.

**Comment by edgarogh** [comment]
Oh I didn't get notified of your comment, I'll fix that asap
Glad that you like it 😉

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -84,6 +87,20 @@
 
         <service android:name=".MainService" />
 
+        <service android:name=".TileMainService"
+            android:exported="true"
+            android:label="@string/app_name"
+            android:icon="@drawable/ic_launcher_foreground"
```
[the remark this answers is not in the document]
Comment (edgarogh) [reply app/src/main/AndroidManifest.xml:93]: Definitely, I noticed the rather small size of the quick tile but didn't realize there was already a bigger icon in the codebase

**Comment by edgarogh** [comment]
So I fixed the icon size, filter to only display connected devices and I updated the strings to "X connected" instead of "X in range" (EN and FR). I merged `master` into my branch but you'll probably rebase/squash anyway.

**Comment by slowscript** [comment]
Thanks. The string saying "in range" was probably fine, but I will leave this up to the translators.

**Pull request merged**